alpha/apps/kaltura/lib/kObjectDeleteHandler.php

```diff
@@ -135,11 +135,20 @@ protected function entryDeleted(entry $entry)
 		
 		$c = new Criteria();
 		$c->add(categoryEntryPeer::ENTRY_ID, $entry->getId());
-		if(!categoryEntryPeer::doSelectOne($c)) {
+		if(categoryEntryPeer::doSelectOne($c)) {
+			kJobsManager::addDeleteJob($entry->getPartnerId(), DeleteObjectType::CATEGORY_ENTRY, $filter);
+		}
+		
+		$ueFilter = new UserEntryFilter();
+		$ueFilter->set("_eq_entry_id", $entry->getId());
+		
+		$c = new Criteria();
+		$c->add(UserEntryPeer::ENTRY_ID, $entry->getId());
+		if(!UserEntryPeer::doSelectOne($c)) {
 			return;
 		}
 		
```

Comment:
Meant something like this ->	
if(UserEntryPeer::doSelectOne($c)) {
		$ueFilter = new UserEntryFilter();
		$ueFilter->set("_eq_entry_id", $entry->getId());
		kJobsManager::addDeleteJob($entry->getPartnerId(), DeleteObjectType::USER_ENTRY, $ueFilter);
		}